UnityProject/Assets/Mirror/Runtime/NetworkIdentity.cs

```diff
@@ -723,6 +728,16 @@ internal void OnStartClient()
                 }
             }
         }
```

Comment:
This could probably go somewhere else, a GameObjectUtils class? Or maybe as an extension of GameObject

Reply:
it has namespace issues since unity is in its own namespace

Reply:
I have no idea what line I commented, I’m on my phone, but I meant the path method

Reply:
mirror*

Reply:
*assembly definition